Update resourcemanager.cpp

Mrpox opened the pull request:
Adding some missing code to make function HTTP.download and HTTP.downloadImage work.
Files were not able to be found in virtual directory "/downloads"

I do believe its 100% needed if ever to build a updater.

Mrpox committed:
Update resourcemanager.cpp

HTTP.download and HTTP.downloadImage now works.
Files can be found in virtual direcotry "/downloads"

mehah commented:
Could you check these errors: https://github.com/mehah/otclient/pull/425/commits/d32016aa4bb2cdce6f0ccb601fc1199487a721bd

- 'g_http': undeclared identifier
-  'dfile': cannot be used before it is initialized
thx

Mrpox committed:
Update resourcemanager.cpp

Fixed following errors:'g_http': undeclared identifier and 'dfile': cannot be used before it is initialized thx

Mrpox commented:
> Could you check these errors: https://github.com/mehah/otclient/pull/425/commits/d32016aa4bb2cdce6f0ccb601fc1199487a721bd
> 
> 
> 
> - 'g_http': undeclared identifier
> 
> -  'dfile': cannot be used before it is initialized
> 
> thx
> 
> 
> 
> 

Fixed
